src/button/tests/unit/Button.ts

```diff
@@ -124,42 +125,42 @@ registerSuite({
 	},
 
 	events() {
-		let blurred = false;
-		let clicked = false;
-		let focused = false;
-		let keydown = false;
-		let keypress = false;
-		let keyup = false;
-		let mousedown = false;
-		let mouseup = false;
+		const onBlur = sinon.spy();
+		const onClick = sinon.spy();
+		const onFocus = sinon.spy();
+		const onKeyDown = sinon.spy();
+		const onKeyPress = sinon.spy();
+		const onKeyUp = sinon.spy();
+		const onMouseDown = sinon.spy();
+		const onMouseUp = sinon.spy();
 
 		widget.setProperties({
-			onBlur: () => { blurred = true; },
-			onClick: () => { clicked = true; },
-			onFocus: () => { focused = true; },
-			onKeyDown: () => { keydown = true; },
-			onKeyPress: () => { keypress = true; },
-			onKeyUp: () => { keyup = true; },
-			onMouseDown: () => { mousedown = true; },
-			onMouseUp: () => { mouseup = true; }
+			onBlur,
+			onClick,
+			onFocus,
+			onKeyDown,
+			onKeyPress,
+			onKeyUp,
+			onMouseDown,
+			onMouseUp
 		});
 
 		widget.sendEvent('blur');
-		assert.isTrue(blurred);
+		assert.isTrue(onBlur.called);
```

Comment:
Can we change these to `calledOnce`, `calledTwice` etc please? Simply using `called` won't catch errors where events are triggered multiple times by mistake.